model/src/test/java/uk/gov/hmcts/payment/api/model/Payment2RepositoryTest.java

```diff
@@ -16,32 +16,32 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
 public class Payment2RepositoryTest {
 
     @Mock
     private Payment2Repository payment2Repository;
 
     private Date fromDate;
     private Date toDate;
-    private PaymentChannel paymentChannel;
+    private PaymentChannel telephonyChannel;
```

Comment:
do we need a test for web type?

Reply:
Not sure that's in the scope for this ticket.

Reply:
That will probably need to be a new ticket, I don't think it's in the scope of this Spring Boot upgrade.